app/assets/stylesheets/active_admin/_header.scss

```diff
@@ -1,156 +1,94 @@
+  }
+
+  h1,
+  .tabs {
     display: table-cell;
     vertical-align: middle;
     white-space: nowrap;
-    color:  $page-header-text-color;
-    margin-right: 20px;
-    margin-bottom: 0px;
-    padding: 3px $horizontal-page-margin 0 $horizontal-page-margin;
-    font-size: 1.3em;
-    font-weight: normal;
-    line-height: 1.2;
-
-    a {
-      text-decoration: none;
-
-      &:hover {
-        color: #fff;
-      }
-    }
-
-    img {
-      position: relative;
-      top: -2px;
-    }
   }
 
-  a, a:link { color: $page-header-text-color; }
-
-  .header-item {
-    top: 2px;
-    position: relative;
-    height: 20px
+  h1 {
+    color: $navbar-color;
+    font-size: 1.3em;
   }
 
-  ul.tabs {
+  .tabs {
     display: table-cell;
-    vertical-align: middle;
-    height: 100%;
     margin: 0;
-    padding: 0;
+    padding: 0 20px;
+    width: 100%;
 
     & > li {
       display: inline-block;
-      margin-right: 4px;
-      margin-top: 5px;
-      margin-bottom: 5px;
-      font-size: 1.0em;
       position: relative;
 
       a {
-        text-decoration: none;
-        padding: 6px 10px 4px 10px;
+        display: inline-block;
+        padding: $navbar-padding-vertical $navbar-padding-horizontal;
         position: relative;
-        @include rounded(10px);
+        transition: background-color 0.15s, color 0.15s;
       }
 
       &.current > a {
-        background: $current-menu-item-background;
-        color: #fff;
-      }
-
-      &.has_nested > a {
-        background: image-url('active_admin/nested_menu_arrow.gif') no-repeat calc(100% - 7px) 50%;
-        padding-right: 20px;
-      }
-
-      &.has_nested.current > a {
-        background: $current-menu-item-background image-url('active_admin/nested_menu_arrow_dark.gif') no-repeat calc(100% - 7px) 50%;
-        padding-right: 20px;
+        background-color: $navbar-link-active-background-color;
+        color: $navbar-link-active-color;
       }
 
       &:hover > a {
-        background: $hover-menu-item-background;
-        color: #fff;
+        background-color: $navbar-link-hover-background-color;
+        color: $navbar-link-hover-color;
       }
 
       &.has_nested:hover > a {
-        @include rounded-top(10px);
-        border-bottom: 5px solid $hover-menu-item-background;
-        background: $hover-menu-item-background image-url('active_admin/nested_menu_arrow_dark.gif') no-repeat calc(100% - 7px) 50%;
-        z-index: 1020;
+        z-index: $z-index-navbar; // so top part of sub menu box-shadow doesn't appear over this link
       }
 
+      &:hover ul {
+        display: block;
+      }
 
-      /* Hover on li, display the ul */
-      &:hover ul { display: block;}
-      /* Drop down menus */
       ul {
-        background: $hover-menu-item-background;
-        @include rounded-all(0,10px,10px,10px);
-        @include shadow(0, 1px, 3px, #444);
-        position: absolute;
-        width: 120%;
-        min-width: 175px;
```

Comment:
@deivid-rodriguez I never thought that menu.add could take a block and then do a sub menu. I thought it was just strictly for links. Thanks for sharing! This will help me account for other uses. Most likely we'll have to document a class you'll have to set here to get the proper alignment.